src/jdk.jpackage/windows/classes/jdk/jpackage/internal/resources/MsiInstallerStrings_en.wxl

```diff
@@ -1,20 +1,20 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This change can be reverted, the English file should not be touched anyways during l10n.